blktap/blktap.cgroups

```diff
@@ -0,0 +1,7 @@
+{
+   "memory": {
+     "MaxPercentageRam": 25.0
+   },
+   "blkio":{
```

Comment:
Does this override the blkio setting in blktap itself? Blktap configures vm.slice for blkio and that is configured
```
[Unit]
Description=VM blkio slice

[Slice]
BlockIOWeight=100

[Install]
WantedBy=multi-user.target
```

Reply:
Ahh Things seem more complicated for some of the cgroups.
blkio has a vm.slice but memory does not.
I could change the behaviour so that we create a tapdisk cgroup underneath any preexisting vm.slice? Or we could create vm.slice for every cgroup ? What do you think?

Reply:
The cgrules now comply with the previous behaviour in addition to moving tapdisks to the Memory cgroup.